packages/rancher-monitoring/rancher-monitoring/generated-changes/patch/templates/prometheus/rules-1.14/kubernetes-system-kube-proxy.yaml.patch

```diff
@@ -10,17 +10,25 @@
  apiVersion: monitoring.coreos.com/v1
  kind: PrometheusRule
  metadata:
-@@ -33,7 +34,7 @@
+@@ -38,11 +39,8 @@
          description: KubeProxy has disappeared from Prometheus target discovery.
          runbook_url: {{ .Values.defaultRules.runbookUrl }}/kubernetes/kubeproxydown
          summary: Target disappeared from Prometheus target discovery.
 -      expr: absent(up{job="kube-proxy"} == 1)
+-      for: {{ dig "KubeProxyDown" "for" "15m" .Values.customRules }}
+-      {{- with .Values.defaultRules.keepFiringFor }}
+-      keep_firing_for: "{{ . }}"
+-      {{- end }}
 +      expr: absent(up{job="{{ include "exporter.kubeProxy.jobName" . }}"} == 1)
-       for: 15m
++      for: 15m
        labels:
-         severity: critical
-@@ -42,3 +43,4 @@
- {{- end }}
- {{- end }}
+         severity: {{ dig "KubeProxyDown" "labelsSeverity" "critical" .Values.customRules }}
+       {{- if or .Values.defaultRules.additionalRuleLabels .Values.defaultRules.additionalRuleGroupLabels.kubeProxy }}
+@@ -54,4 +52,5 @@
+         {{- end }}
+       {{- end }}
  {{- end }}
+-{{- end }}
+\ No newline at end of file
```

Comment:
This `\ No newline at end of file` looks auto generated, and might break helm template?

Reply:
That's just part of the GNu unified diff format: https://www.gnu.org/software/diffutils/manual/html_node/Incomplete-Lines.html